src/bidiMapper/CdpConnection.ts

```diff
@@ -15,25 +15,26 @@
  * limitations under the License.
  */
 
-import {EventEmitter} from '../utils/EventEmitter.js';
+/**
+ * @file CDP interfaces and types that BiDi Mapper expects.
+ */
```

Comment:
Why is file doc preferable over the interface's?

Reply:
/**
 * CDP interfaces that BiDi Mapper expects.
 */

export interface CdpConnection {
  browserClient(): CdpClient;
  getCdpClient(sessionId: string): CdpClient;
}

export interface CdpClient extends EventEmitter<CdpEvents> {
  sendCommand<T extends keyof ProtocolMapping.Commands>(
    method: T,
    ...params: ProtocolMapping.Commands[T]['paramsType']
  ): Promise<ProtocolMapping.Commands[T]['returnType']>;
}


...means that the JSDOC comment is only set to the first interface. But it really applies to both. So, a @fileoverview comment is more appropriate.

In fact, it's @fileoverview, not @file. Will go ahead and fix that too (ref: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler)